bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/pencil-draw-listener/component.jsx

```diff
@@ -117,17 +122,23 @@ export default class PencilDrawListener extends Component {
 
   // main mouse down handler
   mouseDownHandler(event) {
-    if (!this.isDrawing) {
-      window.addEventListener('mouseup', this.mouseUpHandler);
-      window.addEventListener('mousemove', this.mouseMoveHandler, true);
-
-      const { clientX, clientY } = event;
-      this.commonDrawStartHandler(clientX, clientY);
-
-    // if you switch to a different window using Alt+Tab while mouse is down and release it
-    // it wont catch mouseUp and will keep tracking the movements. Thus we need this check.
-    } else {
-      this.sendLastMessage();
+    if (event.button === 0) {
+      if (!this.isDrawing) {
```

Comment:
this doesn't fix any bug, I get the console error and I can delete several annotations yet